src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -43,6 +43,7 @@
         <service id="form.type_guesser.validator" class="Symfony\Component\Form\Extension\Validator\ValidatorTypeGuesser">
             <tag name="form.type_guesser" />
             <argument type="service" id="validator.mapping.class_metadata_factory" />
+            <deprecated>"The \"%service_id%\" service is deprecated since Symfony 3.1 and will be removed in 4.0."</deprecated>
```

Comment:
@HeahDude According to https://github.com/symfony/symfony/blob/master/src/Symfony/Component/DependencyInjection/Tests/Fixtures/xml/services9.xml#L90-L92, it should fail if you escape said `"` and put `"` delimiters though....